swagger.json

```diff
@@ -4187,28 +4187,24 @@
                 "drivers": {
                     "type": "array",
                     "items": {
-                        "type": "object",
-                        "properties": {
-                            "id": {
-                                "type": "integer",
-                                "format": "int64",
-                                "description": "ID of the driver.",
-                                "example": 444
-                            },
-                            "name": {
-                                "type": "string",
-                                "description": "Name of the driver.",
-                                "example": "Fred Jacobs"
+                        "allOf": [
```

Comment:
This is just something to think about, not really sure if it would really cause issues.
```
{
  "drivers": [
    {
      "id": 444,
      "name": "Fred Jacobs"
    }
  ]
}
```
Since this increases the response payload significantly and our API doesn't have pagination it may cause orgs with a lot of drivers get timeouts when they hit this endpoint.

Reply:
Good consideration -- I chatted with @silverspace and the short answer is that we don't need to be concerned about returning the fat driver object here.

We currently use https://datastudio.google.com/reporting/1miK3rbBvE_kvGrck08gquBBxZ3xy-buK/page/4pKV to track metrics.

And I think at some point in the future we'll want to support toggling fat vs skinny response for getting these collection resources via query param.